src/GenFacades.Core.Desktop/GenFacades.Core.Desktop.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
I realize now that this is a bit confusing :laughing: . My reasoning for this was: 1) I wanted the console application to still be called "GenFacades.exe" so that we could still drop it in as a replacement for places still using it and 2) I wanted it to describe the fact that it has the "core" logic of GenFacades. I will think of a better name.